CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.3.11
+  - [#291](https://github.com/logstash-plugins/logstash-input-jdbc/pull/291) Fix a bug where receiving a string value that could not be converted to UTF-8 (such as BLOB data) could crash the input while attempting to log a warning.
```

Comment:
Would you mind reformatting this to meet the [Logstash Changelog formatting guidelines](https://github.com/elastic/logstash/blob/master/CONTRIBUTING.md#what-is-the-format-of-the-changelog-entries)

Reply:
yeah. I was going for internal consistency, but global consistency moving forward is better. :+1: